input/examples/medicationrequest-reaptan.xml

```diff
@@ -4,15 +4,24 @@
     <meta>
         <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-medicationrequest"/>
     </meta>
-    <status value="active"/>
-    <intent value="order"/>
-    <medicationCodeableConcept>
+     <contained>
+    <Medication>
+      <id value="med1"/>
+      <code>
         <coding>
-            <system value="http://snomed.info/sct"/>
-            <code value="926213011000036100"/>
-            <display value="Reaptan 10 mg/10 mg (perindopril arginine/amlodipine) tablet, 10"/>
+          <system value="http://snomed.info/sct"/>
+          <code value="926213011000036100"/>
+          <display value="Reaptan 10 mg/10 mg (perindopril arginine/amlodipine) tablet, 10"/>
```

Comment:
remove display either here or in one of the others. There's a lot of examples that are putting both display and text in CodeabelConcept elements.

Reply:
Done